docs/core/windows-prerequisites.md

```diff
@@ -57,28 +57,51 @@ You can use any editor to develop .NET Core applications using the .NET Core SDK
 
 You can read more about the changes in Visual Studio 2017 in the [release notes](/visualstudio/releasenotes/vs2017-relnotes).
 
-# [.NET Core 2.x](#tab/netcore2x)
+# [.NET Core 2.1](#tab/netcore21)
 
-To develop .NET Core 2.x apps in Visual Studio 2017:
+To develop .NET Core 2.1 apps in Visual Studio 2017:
+
+ 1. [Download and install Visual Studio 2017 version 15.7.0 or higher](/visualstudio/install/install-visual-studio) with the **.NET Core cross-platform development** workload (in the **Other Toolsets** section) selected.
+
+![Screenshot of Visual Studio 2017 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs-15-3-workloads.jpg)
+
+After the **.NET Core cross-platform development** toolset is installed, Visual Studio 2017 15.7 uses .NET Core 2.0 SDK by default. Install the .NET Core 2.1 SDK to get .NET Core 2.1 support in Visual Studio 2017.
```

Comment:
Since Visual Studio 15.8 was released today, should this be updated?

Reply:
Yes, I need to add that! Also need to new screenshots hence the WIP label.